src/os/bluestore/BlueStore.h

```diff
@@ -92,6 +92,12 @@ enum {
   l_bluestore_write_small_wal,
   l_bluestore_write_small_pre_read,
   l_bluestore_write_small_new,
+
+  l_bluestore_cur_ops_in_queue,
+  l_bluestore_cur_bytes_in_queue,
+  l_bluestore_cur_wal_ops_in_wal_queue,
```

Comment:
Nit, you might want to rename this one to `l_bluestore_cur_ops_in_wal_queue` to keep pace with the one below..